src/otx/algo/classification/classifier/base_classifier.py

```diff
@@ -81,7 +81,6 @@ def __init__(
         self.explainer = ReciproCAM(
             self._head_forward_fn,
             num_classes=head.num_classes,
-            optimize_gap=True,
         )
```

Comment:
In my opinion, it seems like a good idea to modify this to accept `optimize_gap` without clearing it. The current change is to change everything that used to be `True` to `False`, and i'm not sure what problems this will cause.

```python
def __init__(
    optimize_gap: bool = True,
):
    self.explainer = ReciproCAM(
        optimize_gap=optimize_gap,
    )
```

Reply:
Fix in [0f91b61](https://github.com/openvinotoolkit/training_extensions/pull/3810/commits/0f91b614d179a84d4476db57a761a6b2f11a8a7f)